homematicip/device.py

```diff
@@ -1423,7 +1423,7 @@ def from_json(self, js):
         super().from_json(js)
         channels = get_functional_channels("MULTI_MODE_INPUT_DIMMER_CHANNEL", js)
         if channels:
-            self.c1dimLevel = channels[1]["dimLevel"]
+            self.c1dimLevel = channels[0]["dimLevel"]
```

Comment:
you could use self.set_attr_from_dict, that way you won't even need to implement __str__